docs/content/templates/variables.md

```diff
@@ -159,6 +159,17 @@ Also available is `.Site` which has the following:
 **.Site.BuildDrafts** A boolean (Default: false) to indicate whether to build drafts. Defined in the site configuration.<br>
 **.Site.Data**  Custom data, see [Data Files](/extras/datafiles/).<br>
 
+## File Variables
+
+The `.File` variable gives you additional information of pages. Available are the following attributes:
```

Comment:
We need to make it clear that the `.File` object is only available on Pages and not Nodes (index, lists).